packages/fxa-content-server/app/scripts/views/settings/emails.js

```diff
@@ -187,7 +186,7 @@ var View = FormView.extend({
       account.unset('originalLoginEmail');
       this.setSearchString('email', email);
       this.displaySuccess(
-        Strings.interpolate(t('Primary email set to %(email)s'), { email }),
+        this.translate(t('Primary email set to %(email)s'), { email }),
```

Comment:
Huh. Do we know why this stopped working? The code looks not recently changed here. Also... do we happen to have a test that could have caught this?

Reply:
> 
> 
> Huh. Do we know why this stopped working? The code looks not recently changed here. 

It is possible that this never worked.  The user has to a) add a secondary email, b) set the secondary as the primary, c) pay attention the notification message, d) know to expect a localized message, and e) report a bug.  It was originally reported in #3258, but it was not fixed in the PR.

> Also... do we happen to have a test that could have caught this?

Off hand I don't remember how many tests we have for testing specific message l10n.  I'll dig around a bit.

Reply:
The view translator actually has a test specific argument (https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/scripts/lib/translator.js#L17) so it won't try to fetch translations.

I updated the view test to ensure that the correct function is called with the correct arguments.